discovery-provider/plugins/network-monitoring/src/metrics/index.ts

```diff
@@ -68,11 +68,14 @@ export const generateMetrics = async (run_id: number) => {
   );
 
   // Record duration for generating metrics and export to prometheus
+  const endTime = Date.now()
   endTimer({ run_id: run_id });
 
   if (userCount > 0) {
     await publishSlackReport({
-      runStartTime: runStartTime.toString(),
+      runStartTimeView: runStartTime.toString(),
+      // duration in milliseconds / (60_000 milliseconds in a minute)
+      runDuration: `${(endTime - runStartTime.getTime()) / 60_000.0} minutes`, 
```

Comment:
given that its several hours, would be more readable to format this into HH:mm:ss? some examples here, unless you have a better way https://stackoverflow.com/questions/19700283/how-to-convert-time-in-milliseconds-to-hours-min-sec-format-in-javascript